commands/issue/board/view/issue_board_view.go

```diff
@@ -120,14 +239,76 @@ func NewCmdView(f *cmdutils.Factory) *cobra.Command {
 			if err := a.SetScreen(screen).SetRoot(root, true).Run(); err != nil {
 				return err
 			}
-
 			return nil
 		},
 	}
 
+	viewCmd.Flags().StringVarP(&opts.AssigneeUsername, "assignee-username", "u", "", "Filter board issues by assignee username")
+	viewCmd.Flags().IntVarP(&opts.AssigneeID, "assignee-id", "i", 0, "Filter board issues by assignee id")
```

Comment:
I'm not super happy about these flags. It would be best to have one single flag to accept the assignee username

Like the flag in `mr creae`: https://github.com/profclems/glab/blob/9b44934c07d1a17d19fd26d5886a752d7cf06a30/commands/mr/create/mr_create.go#L162

You can use the `api.UsersByNames` function as used here:
https://github.com/profclems/glab/blob/9b44934c07d1a17d19fd26d5886a752d7cf06a30/commands/mr/create/mr_create.go#L505-L512